docs/content/tutorials/building-comfort/_index.md

```diff
@@ -0,0 +1,1161 @@
+    dynamically computed, we can use `query-comfort-calc.yaml` file
+    in the directory `devops/drasi`.
+
+```sh
+drasi apply -f query-comfort-calc.yaml
+```
+
+2. To get the alerts for rooms, floors or buildings out of desired
+    comfort-level range, we can use the queries defined in
+    `query-alert.yaml` in the directory `devops/drasi`.
+
+```sh
+drasi apply -f query-alert.yaml
+```
+
+#### SignalR Reaction
+
+Reactions process the stream of query result changes output by one or more
+  Drasi Queries and act on them. You can read
+  [more about Reactions here](https://drasi.io/concepts/reactions/).
+
+To implement a reactive UI, we can use any number of frameworks, and we
+  would need something that can host a websocket for bidirectional
+  communication with the frontend. This should also reliably fall back
+  to ServerSentEvents (SSE) or long polling, ensuring broad support across
+  different network and client configurations. One could use libraries like
+  SignalR, Sockets.IO, Django Channels.
+
+Drasi has a reaction for SignalR available. This simplifies the process, as a
+  simple YAML configuration enables a SignalR hub to push updates from any
+  Drasi query to the UI. Following is the config located in `devops/drasi`
+  directory that lists the queries we're interested in exposing:
+
+```yaml
+apiVersion: v1
+kind: Reaction
+name: building-signalr-hub
+spec:
+  kind: SignalR
+  queries:
+    building-comfort-level-calc:
+    floor-comfort-level-calc:
+    building-alert:
+    room-alert:
+    floor-alert:
+    building-comfort-ui:
+  endpoint:
+    gateway: 8080
+```
+
+In the terminal where you're in directory `devops/drasi`, run the following
+  commands to have a SignalR Reaction deployed to your Drasi instance.
+
+```sh
+drasi apply -f reaction-signalr.yaml
+```
+
+You can expect the following response:
+```
+✓ Apply: Reaction/building-signalr-hub: complete
+```
+
+Note that reactions might take some time to come up. You can use the following
+  convenient command to wait for it (upto 120 seconds) to come up:
+
+```sh
+drasi wait reaction building-signalr-hub -t 120
+```
+
+Once a reaction is online you should be able to see it like so:
+
+```sh
+drasi list reaction
+```
+
+Following should be the output with nothing in the 'Messages' column:
+```
+           ID          | AVAILABLE | MESSAGES  
+-----------------------+-----------+-----------
+  building-signalr-hub | true      |           
+```
+
+##### Troubleshooting the reaction
+Check messages in output of `drasi list reaction`.
+
+For more details you can also use
+  `drasi describe reaction building-signalr-hub`.
+
+Check the logs of pods created for the services `building-signalr-hub-gateway`.
+  These are deployed under drasi's namespace (default: drasi-system) in your
+  kubernetes cluster. You should also investigate the logs of the pods
+  created for source which will have names like
+  `building-facilities-reactivator-xxxxx` or
```

Comment:
These are the pods for the Source, not for the signalR reaction

Reply:
fixed.